api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -791,4 +791,41 @@ public void getCountOfDrugs_shouldReturnTheTotalNumberOfMatchingNumbers() {
 		assertEquals(new Integer(1),
 		    conceptService.getCountOfDrugs(phrase, conceptService.getConcept(conceptId), true, true, true));
 	}
+
+	/**
+	 * @see ConceptServiceImpl#saveConceptProposal(ConceptProposal) 
+	 */
+	@Test
+	public void saveConceptProposal_shouldReturnSavedConceptProposalObject() {
+		String originalText = "OriginalText";
```

Comment:
Did you look at our naming convention for constants? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
My apologies, I really should stop assuming things based on what's already written in the class.

There's the declaration `protected ConceptService conceptService = null;` at the beginning of the class, which I thought was a constant, but it was modified in `runBeforeAllTests` so by definition it does not qualify.